arctic/serialization/numpy_records.py

```diff
@@ -253,7 +254,10 @@ def deserialize(self, item, force_bytes_to_unicode=False):
                 if len(index) and type(index[0]) == bytes:
                     index = index.astype('unicode')
 
-        return Series.from_array(data, index=index, name=name)
+        if PD_VER < '0.23.0':
```

Comment:
Note lexicographic version comparisons don't work right in general for all versions... I guess this is OK for the pandas versions we care about?

Reply:
I tested for 0.22.0, 0.23.x, 0.24.x and 1.x.x and it worked as expected

Reply:
Ok cool. I wonder at some point if we can abstract out the pandas compat code so it doesn't need to exist everywhere, or maybe it'll all go away once we hit 1.x *hopes*

Reply:
if you can get to 0.24.0 these will go away :D